testinfra/modules/package.py

```diff
@@ -74,15 +74,14 @@ class DebianPackage(Package):
 
     @property
     def is_installed(self):
-        return self.run_test((
-            "dpkg-query -f '${Status}' -W %s | "
-            "grep -qE '^(install|hold) ok installed$'"), self.name).rc == 0
+        return self.run_test("dpkg -s %s" % (self.name,)).rc == 0
```

Comment:
This will report uninstalled and non purged packages as installed. A test case is:

```
$ apt-get install redis-server
$ apt-get remove redis-server
$ dpkg -s redis-server >/dev/null
$ echo $?
0
```